next-frontend/package.json

```diff
@@ -43,5 +45,8 @@
     "prettier": "^3.5.3",
     "typescript": "^5"
   },
+  "scarfSettings": {
```

Comment:
what are scraf settings and why do we need to disable them here?

Reply:
It disables automated telemetry gathering. Unfortunately, `package.json` doesn't support comments so I couldn't explain that directly in the file